templates/About.html

```diff
@@ -6,4 +6,121 @@
 
 {% block main %}
 <h1><img src="/static/images/examplePic.png" width="50" height="40" alt="Eventsly">About Us</h1>
+
+<style>
```

Comment:
Styles in an HTML document should be in the head, not in the body. Also it would be better to take these styles out and put them into their own .css file and attach that .css file to the HTML document.